SUILesson7_9/SUILesson7_9/PurchaseView/PurchaseView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  PuchaaseView.swift
+//  SUILesson7_9
+//
+//  Created by Григоренко Александр Игоревич on 23.01.2023.
+//
+
+import SwiftUI
+
+/// Представление экрана с тарифными планами.
+struct PurchaseView: View {
+
+    // MARK: - private constants
+
+    private enum Constants {
+        static let titleName = "Privacy Matters"
+        static let subTitleName = "Protect your online activities with VPN Plus"
+        static let defaultBottomOffset: EdgeInsets = EdgeInsets(top: 0, leading: 0, bottom: 20, trailing: 0)
+        static let imageSize: CGFloat = 120
+        static let imageScaleCoef: CGFloat = 1.4
+        static let zeroOffset: CGFloat = 0
+        static let thirtyOffset: CGFloat = 30
+        static let selectedIndex = 2
+        static let buyName = "Buy"
+        static let eightySize: CGFloat = 80
+        static let fourtySize: CGFloat = 40
+    }
+
+    // MARK: - public properties
+
+    var body: some View {
```

Comment:
```suggestion
    var body: some View {
```
все содержимое разнеси по вычисляемым как у других ребят что б небыли каши